cleanlab/classification.py

```diff
@@ -244,28 +250,28 @@ def fit(
         """This method implements the confident learning. It counts examples
         that are likely labeled correctly and incorrectly and uses their ratio
         to create a predicted confusion matrix.
-        This function fits the classifier (self.clf) to (X, s) accounting for
+        This function fits the classifier (self.clf) to (X, labels) accounting for
         the noise in both the positive and negative sets.
 
         Parameters
         ----------
         X : :obj:`np.array`
           Input feature matrix (N, D), 2D numpy array
 
-        s : :obj:`np.array`
-          A binary vector of labels, s, which may contain mislabeling.
+        labels : :obj:`np.array`
+          A binary vector of labels, labels, which may contain mislabeling.
```

Comment:
why binary

Reply:
old stuff from rankpruning left in here. removed.